src/core/Jalasoft.TeamUp.Projects.Core/ProjectsService.cs

```diff
@@ -25,14 +25,14 @@ public Project GetProject(Guid id)
                 var project = this.projectsRepository.GetById(id);
                 if (project == null)
                 {
-                    throw new ProjectsException(ProjectsException.ProjectsErros.NotFound);
+                    throw new ProjectsException(ProjectsException.ProjectsErrors.NotFound);
```

Comment:
service layer should return null as a valid response, I don't see why it could be considered an exception
the HTTP handler should handle the different scenarios in my opinion, otherwise, we are mixing the responsibilities

Reply:
done